sympy/simplify/sqrtdenest.py

```diff
@@ -236,6 +236,18 @@ def _sqrtdenest0(expr):
         else:
             n, d = [_sqrtdenest0(i) for i in (n, d)]
             return n/d
+
+    if isinstance(expr, Add):
+        cs = []
+        args = []
+        for arg in ordered(expr.args):
```

Comment:
The idea was to find small square roots first, to see if their product can be denested. But maybe it was not that necessary?